cubedash/_stac.py

```diff
@@ -283,11 +283,10 @@ def as_stac_collection(res: CollectionItem) -> pystac.Collection:
         providers=[],
         extent=Extent(
             pystac.SpatialExtent(
-                bboxes=[
-                    res.footprint_wgs84.bounds
-                    if res.footprint_wgs84
-                    else [-180.0, -90.0, 180.0, 90.0]
-                ]
+                # TODO: Find a nicer way to make the typechecker happier
+                # pystac is too specific in wanting a list[float | int]
```

Comment:
It's not going to help right now, but it's possible pystac would accept a PR that changes the type annotation from list to Sequence.